src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+}
+
+.stars--1:nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--2:nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
```

Comment:
Similarly, adjust the selector to `.stars--2 .stars__star:nth-child(-n + 2)` to apply the active image to the first two stars within the `.stars--2` container.